presto-consistent-hash/pom.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
A new module for this seems to expensive to me? May be move under service interface?